src/js/binaryen.js-post.js

```diff
@@ -0,0 +1,29 @@
+return Module;
```

Comment:
I attempted this but was unable to make it work with closure compiler and commonjs/amd environments.